Bottom: 6abf2f310a12e237828c08f12c3cd2c8f4806481
Top:    53e79ddf07237cf5b1aa6c9d33fce5d706701f72
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-04 17:26:52 -0500

Create new play for docker-hosts tasks

The majority of the docker container tasks belong to daedalus, while the
common docker tasks are for all docker hosts.


---
diff --git a/ansible-playbook/production b/ansible-playbook/production
index 07414537..bae6e4f9 100644
--- a/ansible-playbook/production
+++ b/ansible-playbook/production
@@ -1,6 +1,9 @@
 [bacula]
 daedalus.novatech-llc.com
 
+[daedalus]
+daedalus.novatech-llc.com
+
 [docker-hosts]
 daedalus.novatech-llc.com
 
diff --git a/ansible-playbook/site.yml b/ansible-playbook/site.yml
index d5721eb4..f009bdb9 100644
--- a/ansible-playbook/site.yml
+++ b/ansible-playbook/site.yml
@@ -18,7 +18,8 @@
   - role: bacula-script
     assemble_scripts: false
   - role: bacula-catalog
-- name: Install/Setup Docker images and containers
+
+- name: Prepare Docker host systems
   hosts: docker-hosts
   remote_user: ansibleremote
   become: true
@@ -29,6 +30,14 @@
     role: docker-common
     tags:
     - docker_common
+
+- name: Install/Setup Docker images and containers on daedalus
+  hosts: daedalus
+  remote_user: ansibleremote
+  become: true
+  become_user: root
+  become_method: sudo
+  roles:
   - name: Configure Docker networking
     role: docker-networking
     network_connection: '{{ daedalus_net_connection }}'
@@ -83,6 +92,10 @@
 
 - name: Setup the Build System (Test Station) with repo data
   hosts: buildsystem
+  remote_user: ansibleremote
+  become: true
+  become_user: root
+  become_method: sudo
   roles:
   - role: buildsystem
     tags:
diff --git a/ansible-playbook/staging b/ansible-playbook/staging
index e544fd42..ad0ef4ac 100644
--- a/ansible-playbook/staging
+++ b/ansible-playbook/staging
@@ -1,6 +1,9 @@
 [bacula]
 testdaedalus.novatech-llc.com
 
+[daedalus]
+testdaedalus.novatech-llc.com
+
 [docker-hosts]
 testdaedalus.novatech-llc.com
